src/components/Modal/_modals/TransactionConfirmModal/TransactionConfirmModal.tsx

```diff
@@ -82,17 +83,11 @@ export const TransactionConfirmModal: FC<TransactionConfirmModalProps> = ({
             return;
           }
 
-          if (error.response.status === 400) {
-            setErrors(
-              'Transfer was not sent. Probably you filled the form with incorrect data. Please try once again!',
-            );
-            return;
-          }
-
           setErrors(
-            error.response.data
-              ? error.response.data
-              : 'Transfer was not sent. Please verify transfer data and try once again. If problem will happen again, contact with our support.',
+            errorMessage(
+              error.response.data,
+              'Transfer was not sent. Please verify transfer data and try once again. If problem will happen again, contact with our support.',
+            ),
```

Comment:
Maybe it would better to create a variable for `errorMessage` result to reduce nesting?

const errMessage = errorMessage(error.response.data,'Transfer was not sent. Please verify transfer data and try once again. If problem will happen again, contact with our support.')
setErrors(errMessage)

Reply:
Don't agree. in this case it looks ok to me 🙈